doc/manual/embedding.rst

```diff
@@ -29,8 +29,9 @@ We start with a simple C program that initializes Julia and calls some Julia cod
            julia time to cleanup pending write requests
            and run all finalizers
       */
-      jl_atexit_hook();
-      return 0;
+      int ret = 0;
+      jl_atexit_hook(ret);
```

Comment:
I just took it verbatim from https://github.com/JuliaLang/julia/blob/master/examples/embedding.c